data/json/npcs/isherwood_farm/NPC_Eddie_Isherwood.json

```diff
@@ -268,9 +271,9 @@
       "opinion": { "trust": 2, "value": 2 },
       "effect": [
         { "u_add_var": "u_did_eddie_missions", "type": "general", "context": "mission", "value": "yes" },
-        { "u_buy_item": "rope_6", "count": 1 },
+        { "u_spawn_item": "rope_6", "count": 1 },
         { "u_buy_monster": "mon_cow", "count": 1, "name": "Daisy" },
```

Comment:
Wild guess, this "u_buy_monster" may also have the same behavior like "u_buy_item"

Reply:
From looking at the code, yes. Even though it's free it'll fail if you owe money.

Reply:
Added a C++ fix for that.